opm/simulators/flow/GenericCpGridVanguard.cpp

```diff
@@ -438,7 +438,7 @@ void GenericCpGridVanguard<ElementMapper,GridView,Scalar>::doCreateGrids_(Eclips
     if (isRoot) {
         input_grid = &eclState.getInputGrid();
         global_porv = eclState.fieldProps().porv(true);
-        OpmLog::info("\nProcessing grid");
+        OpmLog::info("Processing grid");
```

Comment:
I think I'd like to keep this newline, please.  It makes the 'Processing' stage stand out a bit.

Reply:
Ok, this is now as before.